docs/examples/ingestion/redis_ingestion_pipeline.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>so if we  add </p><p><code></code><code>!echo "This is a test file: two!" &gt; test_redis_data/test2.txt</code><code></code></p><p>again.</p><p>It will upsert test2 right bc timestamp</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
yea, most likely 🤷🏻‍♂️ It's not easy to fix though, because the metadata is just a keyval dict, and the timestamp could technically be under any key

And since metadata is used when text splitting, if the metadata actually DID change (not just a timestamp), it can change how the text is split. 

Feels quite messy to account for this, maybe better to leave it naive for now :)